tests/cli/cli_tests.js

```diff
@@ -276,24 +292,18 @@ describe('cli', function () {
+    const encodedRuleset = 'ewogICAgIiRzY2hlf'
```

Comment:
Errmm. Can you double check this is good - feels like a bit of a big change relative to the log message.

Reply:
I'll add a different test in a different set for this.
During more follow up testing I realize that the original design of this test is flawed.
* This is string is not a base64 string, repolinter is going to treat it as a file name.
* But *THIS* string is way too long to resolve in dir structure, so instead of expecting "encoded string" is wrong, linter is complaining "file name is too long to resolve" as "Error: ENAMETOOLONG: name too long".

I'll put the "name too long" as a different test somewhere else here.

Reply:
The previous encodedRuleset is a invalid base64, so instead of a VERY long invalid base64 string, I can provide any string here, which will be interpreted as file name...
Because the linter's ruleset is too overloaded (as rulesetUrl, rulesetPath and encoded in index.js), I don't have a good solution to pick that apart unless creating another overloading interface to accept an object.
I need to think more on that approach to refactor lint overall structure.